mappings/net/minecraft/client/texture/NativeImage.mapping

```diff
@@ -27,20 +72,48 @@ CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 		ARG 2 width
 		ARG 3 height
 		ARG 5 pointer
+	METHOD <init> (Lnet/minecraft/class_1011$class_1012;IIZ)V
+		ARG 2 width
+		ARG 3 height
+	METHOD <init> (Lnet/minecraft/class_1011$class_1012;IIZJ)V
+		ARG 2 width
+		ARG 3 height
+		ARG 5 glPointer
 	METHOD method_15877 writeFile (Ljava/lang/String;)V
-	METHOD method_15990 fromBase64 (Ljava/lang/String;)Lnet/minecraft/class_1011;
+		ARG 1 fileName
+	METHOD method_15990 read (Ljava/lang/String;)Lnet/minecraft/class_1011;
 		ARG 0 string
 	METHOD method_4300 resizeSubRectTo (IIIILnet/minecraft/class_1011;)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 width
+		ARG 4 height
+		ARG 5 targetImage
 	METHOD method_4301 upload (IIIZ)V
+		ARG 1 level
+		ARG 2 xOffset
+		ARG 3 yOffset
 	METHOD method_4302 untrack ()V
-	METHOD method_4303 fromByteBuffer (Lnet/minecraft/class_1011$class_1012;Ljava/nio/ByteBuffer;)Lnet/minecraft/class_1011;
+	METHOD method_4303 read (Lnet/minecraft/class_1011$class_1012;Ljava/nio/ByteBuffer;)Lnet/minecraft/class_1011;
+	METHOD method_4304 copyRect (IIIIIIZZ)V
+		ARG 1 srcX
+		ARG 2 srcY
+		ARG 3 dstX
+		ARG 4 dstY
```

Comment:
I suggest "rectangle", "source" and "destination" (or "target"). Mre abbrs makes rding the code take more effrt.

Reply:
I don't see anything wrong with the current name. Also what is `target`? Is that the source or the destination?